### ISSUE TITLE
Lack of Tab Completion at Eclipse REPL

### ISSUE THREAD

**Issue by ranjanan** [JuliaComputing/JuliaDT#15]
It would be nice if there were tab completion at the Eclipse REPL. It would allow for autocompletion as well as `LateX` support

**Comment by aviks** [comment]
This is going to be low priority.

**Comment by scidom** [comment]
Apart from the Eclipse REPL, it would be nice to have tab completion in the text editor too for julia files.

**Comment by ViralBShah** [comment]
@MikeInnes Does Atom have correctly scoped tab completion?

**Comment by ViralBShah** [comment]
@aviks @snefru Can we do tab completion of all the stuff in Base or something along those lines?

**Comment by MikeInnes** [comment]
Yes, if you're referring to module scoping. e.g. typing while inside module `Foo` gets you functions defined in Base + Foo, and typing `Foo.` gives you completions defined in Foo only.
